python/symbolica_community/examples/physics/vakint.py

```diff
@@ -0,0 +1,92 @@
+from symbolica_community import Vakint, VakintEvaluationMethod, VakintExpression, VakintNumericalResult
+import symbolica_community as sb
+from symbolica_community import E
+
+masses = {"muvsq": 2., "mursq": 3.}
```

Comment:
Do you want to keep the mass as a string instead of a Symbolica symbol?

Reply:
I did not change this in `vakint` (it'd be right to do it but annoying and not so important now).
But I did change it in the vakint python API exposed to the user.

I also migrated both `vakint` and `symbolica-community` to Symbolica `0.12.1`, so it should be ready for merge.